docker-compose.yml

```diff
@@ -13,16 +13,15 @@ services:
       - 5672:5672
 
   worker:
-    build: .
-    command: celery -A whistleblower.tasks worker -B
+    image: serenata/whistleblower
     restart: always
     depends_on:
       - mongo
       - rabbitmq
     env_file:
       - .env
     volumes:
-      - ./data:/usr/src/app/data
+      - /tmp/serenata-data:/usr/src/app/data
```

Comment:
This change is because of the main repository, okfn-brasil/serenata-de-amor, started saving the data into this new folder, not `./data` anymore. For this pull request, is not strictly needed, but maintains the same pattern across services. It also eases the process of merging the repositories, once this happens.